dev/VSIX/ProjectTemplates/Desktop/CSharp/UnitTestApp/Properties/PublishProfiles/win10-arm64.pubxml

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--
+https://go.microsoft.com/fwlink/?LinkID=208121.
+-->
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PublishProtocol>FileSystem</PublishProtocol>
+    <Platform>ARM64</Platform>
+    <RuntimeIdentifier>win10-arm64</RuntimeIdentifier>
+    <PublishDir>bin\$(Configuration)\$(TargetFramework)\$(RuntimeIdentifier)\publish\</PublishDir>
+    <SelfContained>true</SelfContained>
+    <PublishSingleFile>False</PublishSingleFile>
+    <PublishReadyToRun Condition="'$(Configuration)' == 'Debug'">False</PublishReadyToRun>
+    <PublishReadyToRun Condition="'$(Configuration)' != 'Debug'">True</PublishReadyToRun>
+   <!-- 
```

Comment:
you can remove this (all other templates should also be updated) - CsWinRT now supports PublishTrimmed.  You could also set it based on Configuration for faster inner loop, as with PublishReadyToRun.  [Gallery does this](https://github.com/microsoft/WinUI-Gallery/blob/861564349b526edada16b51ff6b78dd105564fa5/WinUIGallery/Properties/PublishProfiles/win10-x64.pubxml#L16).

Reply:
I'll update this and also the other project templates.